capstone-backend/src/test/java/com/google/sps/data/ClubTest.java

```diff
@@ -10,7 +10,19 @@
 public final class ClubTest {
   private Club club;
 
-  private static final String NAME = "name";
+  private static final String ORIGINAL_NAME = "originalName";
+  private static final String NEW_NAME = "newName";
+
+  private static final String ORIGINAL_DESCRIPTION = "originalDescription";
+  private static final String NEW_DESCRIPTION = "newDescription";
+
+  private static final String ANNOUNCEMENT = "announcement";
+
+  private static final Assignment ASSIGNMENT_ONE = new Assignment("one", "id", "created", "due");
+  private static final Assignment ASSIGNMENT_TWO = new Assignment("two", "id", "created", "due");
+
+  private static final String ORIGINAL_OWNER = "originalOwner";
+  private static final String NEW_OWNER = "newOwner";
```

Comment:
```
ORIGINAL_NAME
ORIGINAL_DESCRIPTION
ORIGINAL_OWNER
```
Are never used. Consider removing them.

Reply:
Similar to below comment, pulled this out in the last commit which didn't push. Thanks for catching this.